Makefile

```diff
@@ -4,7 +4,7 @@ all: binary
 binary: clean
 	@echo "PHASE: Building sgx-device-plugin ... "
 	mkdir _output/
-	GOOS=linux go build -o _output/sgx-device-plugin ./cmd/sgx-device-plugin/*.go
+	GOOS=linux go build -tags disable_aesm -o _output/sgx-device-plugin ./cmd/sgx-device-plugin/*.go
```

Comment:
I suggest using a common name as "disable_mount_points",  aesmd socket is just the first one as far as now.  What's more, how about the logic about returning mount points as a formal code, just let the mount points items in the list to be configurable.

Reply:
We consider to use cmdline option --enable-aesm-socket-attach and --aesm-socket at this stage. Let me know if any comment.